src/components/nav-bar/NavBar.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import {NavLink} from "react-router-dom";
+
+const NavBar: React.FC = () => {
+    return (
+        <nav>
+            <div className="nav-wrapper">
+                <NavLink to="/" className="brand-logo center">Trip Viewer</NavLink>
+                <ul className="right">
+                    <li><NavLink to="/" exact>Home</NavLink></li>
```

Comment:
why do you use NavLink instead of Link?

Reply:
With NavLink I can customize active class for active link.